doc/research/paf24/general/leaderboard_summary.md

```
@@ -0,0 +1,119 @@
+    - The ego-vehicle must cross a lane of moving traffic to exit the highway at an off-ramp.
+      ![image](https://leaderboard.carla.org/assets/images/TR21.png)
+    - The ego-vehicle is approached by an emergency vehicle coming from behind.
+      ![image](https://leaderboard.carla.org/assets/images/TR23.png)
+  - ### **Obstacle avoidance:**
+    - The ego-vehicle encounters an obstacle blocking the lane and must perform a lane change into traffic moving in the same or opposite direction to avoid it. The obstacle may be a construction site, an accident or a parked vehicle.
+      ![image](https://leaderboard.carla.org/assets/images/TR14.png)
+    - The ego-vehicle encounters a parked vehicle opening a door into its lane and must maneuver to avoid it.
+      ![image](https://leaderboard.carla.org/assets/images/TR15.png)
+    - The ego-vehicle encounters a slow moving hazard (e.g. bicycles) blocking part of the lane. The ego-vehicle must brake or maneuver next to a lane of traffic moving in the same or opposite direction to avoid it.
+      ![image](https://leaderboard.carla.org/assets/images/TR16.png)
+    - The ego-vehicle encounters an oncoming vehicles invading its lane on a bend due to an obstacle.
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)